build.sbt

```diff
@@ -85,6 +85,23 @@ testScalastyle := scalastyle.in(Test).toTask("").value
 
 (test in Test) := ((test in Test) dependsOn testScalastyle).value
 
+/*********************
+ *  MIMA settings    *
+ *********************/
+
+(test in Test) := ((test in Test) dependsOn mimaReportBinaryIssues).value
+
+def getVersion(version: String): String = {
+    version.split("\\.").toList match {
+        case major :: minor :: rest => s"$major.$minor.0" ;
+        case _ => throw new Exception("Could not find previous version")
```

Comment:
Add the current version and the split list in the exception message so that it easy to see why it failed.
Also print a line saying what the previous version for mima was computer to be so that it's easier to correlate when there is download failure.

Reply:
Also remove the semicolon